AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -232,9 +232,9 @@ - (void)testCleanUpOnDisabling {
   [[MSAuthTokenContext sharedInstance] setAuthTokenHistory:authTokenHistory];
   [[MSAuthTokenContext sharedInstance] setAuthToken:fakeToken withAccountId:fakeAccountId expiresOn:nil];
   [self.sut setEnabled:YES];
-  id accountMock = OCMPartialMock([MSALAccount new]);
+  id accountMock = OCMClassMock(MSALAccount.class);
```

Comment:
Add stopMocking call for it

Reply:
My understanding is we don't have to stop mocking ourselves if not stubbing class methods. But if it's confusing or seems error prone or I'm wrong I can add the stop mocking.

Reply:
I am trying to keep things simple: Do I mock class methods YES > stop mocking  otherwise don't. But I can be wrong.

Reply:
Need to call stop mocking whenever it uses class mock. It will affect initialize when it instantiate it

Reply:
I am not so sure about that. To me the OCMClassMock just subclass and return an instance of that subclass, nothing really bad. It's only when you stub a class method that OCMMock creates a meta-class and substitute the meta class of the mocked class. Only at that point it can get bad if the mock stubbing the class method is not explicitly stopped or deallocated.